src/plugin.js

```diff
@@ -2,6 +2,11 @@ import videojs from 'video.js';
 import window from 'global/window';
 import document from 'global/document';
 
+/** THe logic below is used to check if flash is disabled or not installed
+ *  in IE browser and show the appropriate message to the user.
+ */
```

Comment:
We shouldn't use these docblock comments unless we want this variable in generated API docs (which we don't since it's private). Use `//` comments instead.